terraform/ipfs-server/process-stream.sh

```diff
@@ -8,11 +8,14 @@ IPFS_GATEWAY=http://$DOMAIN_NAME:8080
 
 cd ~/live
 
-what='LIVE'
-rm -rf $what*.ts
-rm -rf $what.m3u8
+what=`date +Y%m%d%H%M`
+what="LIVE-$what"
```

Comment:
Can't this be done in a single line?

Reply:
Probably... didn't want to risk it.. + not 100% sure how.
probobly a $(date...)-$what